src/ray/core_worker/core_worker.cc

```diff
@@ -1025,6 +1025,7 @@ void CoreWorker::PutObjectIntoPlasma(const RayObject &object, const ObjectID &ob
 }
 
 void CoreWorker::PromoteObjectToPlasma(const ObjectID &object_id) {
+  // TODO(swang): Remove.
```

Comment:
?

Reply:
Ah I meant to remove this codepath because it won't be used anymore, but just wanted to check if CI would pass first.

Reply:
If it's okay with you, I might leave this as a follow-up PR to unblock the release. Looks like this is codepath is used by Java too.